packages/cli/src/domains/magento2/docker/installMagentoImage.ts

```diff
@@ -0,0 +1,81 @@
+import { spawn } from 'child_process';
+import removeDockerContainer from './removeDocker';
+import { logSimpleErrorMessage } from '../functions/terminalHelpers';
+
+import { note, spinner } from '@clack/prompts';
+import picocolors from 'picocolors';
+import { t } from 'i18next';
+
+/** Handles Magento 2 Docker Image installation */
+const installMagentoImage = async (
+  magentoDirName: string,
+  magentoDomainName: string
+): Promise<any> => {
+  // const command = `curl -s https://raw.githubusercontent.com/markshust/docker-magento/master/lib/onelinesetup | bash -s -- ${magentoDomainName} 2.4.4`;
+  const options = {
+    cwd: magentoDirName
+  };
+
+  const sp = spinner();
+
+  return new Promise((resolve) => {
+    const curl = spawn(
+      'curl',
+      [
+        '-s',
+        'https://raw.githubusercontent.com/markshust/docker-magento/master/lib/onelinesetup'
+      ],
+      options
+    );
+    const bash = spawn(
+      'bash',
+      ['-s', '--', magentoDomainName, '2.4.4'],
```

Comment:
I believe Magento version should be configurable, with a default that is the latest version (so it needs to be automated, or manually updated a few times a year)

But... the underlying API is already handling that, so I guess you can just remove the version from here.

Reply:
I'm not so sure about that 
```# Run this automated one-liner from the directory you want to install your project.
curl -s https://raw.githubusercontent.com/markshust/docker-magento/master/lib/onelinesetup | bash -s -- magento.test 2.4.5-p1 community```

https://github.com/markshust/docker-magento - they are using a newer version, but our docs are still mentioning 2.4.4 - can't guarantee that newer works too. 

Need to test